packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -95,11 +95,14 @@ of the iOS app that is using this module.''',
         help: 'Validate but do not upload the patch.',
       )
       ..addOption(
-        CommonArguments.privateKeyArgName,
+        CommonArguments.privateKeyArg.name,
         hide: true,
-        help: '''
-The path for a private key file that will be used to sign the patch artifact.
-''',
+        help: CommonArguments.privateKeyArg.description,
+      )
+      ..addOption(
```

Comment:
I am not completely sure I get what you mean about flavor.

But maybe renaming this to `Arguments` instead of `CommonArguments` would be a good call yes, right now it is a bit of a mouthful